site/content/integrate/apps/drivers/golang.md

```diff
@@ -3,6 +3,436 @@ title: Golang
 description: The golang App driver
 weight: 10
 ---
+The following sections assume you've created a new, empty Golang project. 
```

Comment:
Yes, I agree with @mickmister, I think it's important to tell them that step. If this were a Node project, we'd probably also manually tell them to `npm init` because that signifies to the reader what we're about to do. I think we should do the same here.